client/new-stack-editor/lib/views/flexsplit/resizer.coffee

```diff
@@ -0,0 +1,122 @@
+kd = require 'kd'
+FlexSplit = require './index'
+
+
+module.exports = class FlexSplitResizer extends kd.View
+
+  constructor: (options = {}, data) ->
+
+    options.type     ?= FlexSplit.HORIZONTAL
+    options.cssClass  = 'flex-resizer'
+    options.draggable = { axis: options.type.axis }
+
+    super options, data
+
+    @views      = []
+    @_fractions = []
+    { @type, view } = @getOptions()
+
+    @addView view
+
+    @on 'DragFinished', @dragFinished
+    @on 'DragStarted',  @dragStarted
+
+
+  _getViewIndex: (view) ->
+    if view is @views[0] then 0 else 1
+
+
+  addView: (view) ->
+
+    @views.push view
+
+    view.on FlexSplit.EVENT_EXPAND, =>
+
+      return  if view.hasClass 'expanded'
+      view.setClass 'expanded'
```

Comment:
I thought about this again and kept it as is for now. Since I don't need any specific class to be as view, I don't want to create another dependency. Right now you can use and `kd.View` as `view` but if I require something like this it will add more over head than it's solving. `expand` class is dynamically added and it's ok if someone removes it, it will just re-trigger expand if they don't handle this status on their own, it won't break anything in `FlexSplit`.